definitions/velocity.proto

```diff
@@ -32,3 +32,13 @@ message Sample {
   required double timestamp   = 1;
   repeated float values       = 2;
 }
+
+message DetectMotionRequest {
```

Comment:
I think, response is missing as well, unless we are going to send response in different format.